pkg/apis/core/validation/validation.go

```diff
@@ -3165,6 +3167,10 @@ func validateContainerCommon(ctr *core.Container, volumes map[string]core.Volume
 		allErrs = append(allErrs, field.Required(path.Child("image"), ""))
 	}
 
+	_, err := dockerref.ParseNormalizedNamed(ctr.Image)
+	if err != nil {
+		allErrs = append(allErrs, field.Invalid(path.Child("image"), ctr.Image, "fails dockerref regular expressions"))
```

Comment:
we need a better error message - following the usual "must X"  and "may not Y" format.  This may mean cloning that logic so that we can produce those messages.

Reply:
Done.